### PR TITLE
IDEX-3138. Simplify choices for Che project creation wizard

### PR DESCRIPTION
**Pull request by RomanNikitenko**
Remove TutorialProjectType.
Add tooltip for artifact ID and group ID.
Fix NPE in project wizard.

### CODE REVIEW

File: plugin-java/che-plugin-java-ext-maven/src/main/java/org/eclipse/che/ide/extension/maven/client/wizard/MavenPagePresenter.java [hunk 1]
```diff
@@ -284,10 +284,12 @@ private void validateCoordinates() {
...
         List<String> values = attributes.get(attrId);
-        if (!(values == null || values.isEmpty())) {
-            return values.get(0);
+        if (values == null || values.isEmpty()) {
+            return "";
         }
-        return "";
+
+        String value = values.get(0);
+        return value != null ? value : "";
```
Comment (evoevodin) [inline plugin-java/che-plugin-java-ext-maven/src/main/java/org/eclipse/che/ide/extension/maven/client/wizard/MavenPagePresenter.java]: `MoreObjects.firstNonNull(values.get(0), "")` see [doc](http://docs.guava-libraries.googlecode.com/git/javadoc/com/google/common/base/MoreObjects.html#firstNonNull(T,%20T))

File: plugin-sdk/che-plugin-sdk-ext-plugins/src/main/java/org/eclipse/che/ide/ext/tutorials/client/wizard/ExtensionPagePresenter.java [hunk 2]
```diff
@@ -196,10 +196,12 @@ private void validateCoordinates() {
...
         List<String> values = attributes.get(attrId);
-        if (!(values == null || values.isEmpty())) {
-            return values.get(0);
+        if (values == null || values.isEmpty()) {
+            return "";
         }
-        return "";
+
+        String value = values.get(0);
+        return value != null ? value : "";
```
Comment (evoevodin) [inline plugin-sdk/che-plugin-sdk-ext-plugins/src/main/java/org/eclipse/che/ide/ext/tutorials/client/wizard/ExtensionPagePresenter.java]: the same [comment](https://github.com/codenvy/che-plugins/pull/303/files#r41338746)

File: plugin-sdk/che-plugin-sdk-ext-plugins/src/main/java/org/eclipse/che/ide/ext/tutorials/client/wizard/ExtensionPagePresenter.java [hunk 3]
```diff
@@ -196,10 +198,10 @@ private void validateCoordinates() {
     private String getAttribute(String attrId) {
         Map<String, List<String>> attributes = dataObject.getProject().getAttributes();
         List<String> values = attributes.get(attrId);
-        if (!(values == null || values.isEmpty())) {
-            return values.get(0);
+        if (values == null || values.isEmpty()) {
+            return "";
         }
-        return "";
+        return MoreObjects.firstNonNull(values.get(0), "");
```
Comment (sleshchenko) [inline plugin-sdk/che-plugin-sdk-ext-plugins/src/main/java/org/eclipse/che/ide/ext/tutorials/client/wizard/ExtensionPagePresenter.java]: I think it is better to use static import here

**Comment by skabashnyuk** [comment]
LGTM

**Comment by vparfonov** [comment]
ok

**Pull request merged**